### PR TITLE
Update code contribution page

### PR DESCRIPTION
**Pull request by U8NWXD**
This PR depends on #2 and #3 because it introduces links to the pages added by those PRs

### CODE REVIEW

File: Contributing-code-to-Oppia.md
Comment (seanlip) [inline Contributing-code-to-Oppia.md:5]: Missing period at end of sentence.

File: Contributing-code-to-Oppia.md
Comment (seanlip) [inline Contributing-code-to-Oppia.md:11]: Add comma after "If you do not sign the CLA"?

File: Contributing-code-to-Oppia.md
Comment (seanlip) [inline Contributing-code-to-Oppia.md:162]: "the that" seems wrong.

File: Contributing-code-to-Oppia.md
Comment (seanlip) [inline Contributing-code-to-Oppia.md:164]: Add comma after "will pass the tests" (or restructure the sentence so there's no ambiguity on how to parse it)

File: Contributing-code-to-Oppia.md
Comment (seanlip) [inline Contributing-code-to-Oppia.md:171]: Maybe give some guidance on when to ask? Or is it not necessary?

File: Contributing-code-to-Oppia.md
Comment (seanlip) [inline Contributing-code-to-Oppia.md:173]: Should "get help" be "getting help"?

File: Contributing-code-to-Oppia.md
Comment (seanlip) [inline Contributing-code-to-Oppia.md:181]: change in --> changing

**Review by seanlip — CHANGES_REQUESTED** [review]

File: Contributing-code-to-Oppia.md
Comment (U8NWXD) [reply Contributing-code-to-Oppia.md]: done

File: Contributing-code-to-Oppia.md
Comment (U8NWXD) [reply Contributing-code-to-Oppia.md]: done

File: Contributing-code-to-Oppia.md
Comment (U8NWXD) [reply Contributing-code-to-Oppia.md]: done

File: Contributing-code-to-Oppia.md
Comment (U8NWXD) [reply Contributing-code-to-Oppia.md]: done

File: Contributing-code-to-Oppia.md
Comment (U8NWXD) [reply Contributing-code-to-Oppia.md]: done

File: Contributing-code-to-Oppia.md
Comment (U8NWXD) [reply Contributing-code-to-Oppia.md]: done

File: Contributing-code-to-Oppia.md
Comment (U8NWXD) [reply Contributing-code-to-Oppia.md]: done

**Comment by U8NWXD** [comment]
@seanlip I've addressed your comments PTAL

File: Contributing-code-to-Oppia.md
Comment (seanlip) [inline Contributing-code-to-Oppia.md:93]: Typo in "typescript"

File: Contributing-code-to-Oppia.md
Comment (seanlip) [inline Contributing-code-to-Oppia.md:79]: I thought this was being done as a gsoc project; maybe confirm with @nithusha21 @kevintab95 whether it should still be offered.

File: Contributing-code-to-Oppia.md
Comment (seanlip) [inline Contributing-code-to-Oppia.md:99]: rachelwchen --> mschanteltc

File: Contributing-code-to-Oppia.md
Comment (seanlip) [inline Contributing-code-to-Oppia.md:189]: Then, only ... --> Only after getting that trial PR merged (or at least approved by all reviewers) should you do the full change.

(This is an attempt to have the sentence describe events in chronological order.)

File: Contributing-code-to-Oppia.md
Comment (seanlip) [inline Contributing-code-to-Oppia.md:193]: Is it worth explaining how to do this in the GitHub UI too? It's kind of nice to see who last changed a specific line, so that might be even more useful.

File: Installing-Oppia-(Linux;-Python-3).md
Comment (seanlip) [inline Installing-Oppia-(Linux;-Python-3).md:7]: Add comma after Oppia
some --> a

File: Installing-Oppia-(Linux;-Python-3).md
Comment (seanlip) [inline Installing-Oppia-(Linux;-Python-3).md:34]: I think "highly recommend" is more common.

File: Installing-Oppia-(Linux;-Python-3).md
Comment (seanlip) [inline Installing-Oppia-(Linux;-Python-3).md:80]: is on --> is running

**Review by seanlip — CHANGES_REQUESTED** [review]
This should be my last round of comments on this PR.

File: Installing-Oppia-(Linux;-Python-3).md
Comment (U8NWXD) [reply Installing-Oppia-(Linux;-Python-3).md]: This file change isn't actually part of this PR. I just forgot to push the latest `master` from `oppia/oppia` to the `master` of `U8NWXD/oppia-wiki`. I've fixed this now

File: Installing-Oppia-(Linux;-Python-3).md
Comment (U8NWXD) [reply Installing-Oppia-(Linux;-Python-3).md]: This file change isn't actually part of this PR. I just forgot to push the latest `master` from `oppia/oppia` to the `master` of `U8NWXD/oppia-wiki`. I've fixed this now

File: Installing-Oppia-(Linux;-Python-3).md
Comment (U8NWXD) [reply Installing-Oppia-(Linux;-Python-3).md]: This file change isn't actually part of this PR. I just forgot to push the latest `master` from `oppia/oppia` to the `master` of `U8NWXD/oppia-wiki`. I've fixed this now

File: Contributing-code-to-Oppia.md
Comment (U8NWXD) [reply Contributing-code-to-Oppia.md:193]: done

File: Contributing-code-to-Oppia.md
Comment (U8NWXD) [reply Contributing-code-to-Oppia.md]: done

File: Contributing-code-to-Oppia.md
Comment (U8NWXD) [reply Contributing-code-to-Oppia.md]: done

File: Contributing-code-to-Oppia.md
Comment (U8NWXD) [reply Contributing-code-to-Oppia.md]: done

File: Contributing-code-to-Oppia.md
Comment (U8NWXD) [reply Contributing-code-to-Oppia.md:79]: I think this should no longer be a starter task, but I'm checking with the GSoC mentors. If they agree, I'll remove this line

**Comment by U8NWXD** [comment]
@seanlip I've addressed your comments. PTAL

File: Contributing-code-to-Oppia.md
Comment (U8NWXD) [reply Contributing-code-to-Oppia.md]: I've removed it as a starter issue after conferring with @nithusha21

**Review by seanlip — APPROVED** [review]
LGTM from my end, thanks @U8NWXD.

@DubeySandeep -- could you please do a sanity check on this as dev workflow lead?

**Review by DubeySandeep — APPROVED** [review]
LGTM!

@U8NWXD Can we add the TOC on the top of the wiki page? Let's merge this once this comment is resolved!

**Pull request merged**
